### PR TITLE
SEO-171714 - Window Forms Page with redirect  

### PR DESCRIPTION
**Pull request by SellineRita**
HI @Aishwarya-Ganesan ,
Please review changes for Window Forms redirect links.

Regards,
Selline Rita.

### CODE REVIEW

File: WindowsForms/Grid-Control/Cell-Style-Architecture.md [hunk 1]
```diff
@@ -612,7 +612,7 @@ N> <kbd>Ctrl</kbd>+<kbd>Delete</kbd> key combination can be used to clear the se
 
 ### Clearing Only the Data
 
-For clearing only the selected range of cells, make use of the [Model.Clear](https://help.syncfusion.com/cr/windowsforms/Syncfusion.Windows.Forms.Grid.GridModel.html#Syncfusion_Windows_Forms_Grid_GridModel_Clear_System_Boolean_) method. This method clears only the cells that are selected. 
+For clearing only the selected range of cells, make use of the [Model.Clear](https://help.syncfusion.com/cr/windowsforms/Syncfusion.Windows.Forms.Grid.GridStyleInfo.html#Syncfusion_Windows_Forms_Grid_GridStyleInfo_ModifyStyle_Syncfusion_Styles_IStyleInfo_Syncfusion_Styles_StyleModifyType_) method. This method clears only the cells that are selected.
```
Comment (gowthamanthavasiyappan) [inline WindowsForms/Grid-Control/Cell-Style-Architecture.md:615]: Recheck this link.

**Review by gowthamanthavasiyappan — CHANGES_REQUESTED** [review]
Commented

File: WindowsForms/DataGrid/InteractiveFeatures.MD [hunk 2]
```diff
@@ -136,7 +136,7 @@ Me.sfDataGrid1.GroupCaptionContextMenu.Items.Add("Collapse", Nothing, OnCollapse
 
 ### ContextMenu for GroupSummaryRow
 
-The context menu for the group summary can be set by using [SfDataGrid.GroupSummaryContextMenu](https://help.syncfusion.com/cr/windowsforms/Syncfusion.WinForms.DataGrid.SfDataGrid.html#Syncfusion_WinForms_DataGrid_SfDataGrid_GroupSummaryContextMenu) property.
+The context menu for the group summary can be set by using [SfDataGrid.GroupSummaryContextMenu](https://help.syncfusion.com/cr/windowsforms/Syncfusion.WinForms.DataGrid.SfDataGrid.html#Syncfusion_WinForms_DataGrid_SfDataGrid_GroupCaptionContextMenu) property.
```
Comment (Girija-Rajendran) [inline WindowsForms/DataGrid/InteractiveFeatures.MD:139]: Is this correct?

File: WindowsForms/DataGrid/InteractiveFeatures.MD [hunk 3]
```diff
@@ -650,7 +650,7 @@ Me.sfDataGrid1.Columns("OrderID").ShowToolTip = True
 {% endhighlight %}
 {% endtabs %}
 
-N>[GridColumnBase.ShowToolTip](https://help.syncfusion.com/cr/windowsforms/Syncfusion.WinForms.DataGrid.GridColumnBase.html#Syncfusion_WinForms_DataGrid_GridColumnBase_ShowToolTip) takes higher priority than [SfDataGrid.ShowToolTip](https://help.syncfusion.com/cr/windowsforms/Syncfusion.WinForms.DataGrid.SfDataGrid.html#Syncfusion_WinForms_DataGrid_SfDataGrid_ShowToolTip).
+N>[GridColumnBase.ShowToolTip](https://help.syncfusion.com/cr/windowsforms/Syncfusion.WinForms.DataGrid.SfDataGrid.html#Syncfusion_WinForms_DataGrid_SfDataGrid_ShowToolTip) takes higher priority than [SfDataGrid.ShowToolTip](https://help.syncfusion.com/cr/windowsforms/Syncfusion.WinForms.DataGrid.GridColumnBase.html#Syncfusion_WinForms_DataGrid_GridColumnBase_ShowToolTip).
```
Comment (Girija-Rajendran) [inline WindowsForms/DataGrid/InteractiveFeatures.MD:653]: Recheck this link

File: WindowsForms/Grid-Control/Cell-Style-Architecture.md [hunk 4]
```diff
@@ -326,7 +326,7 @@ gridControl1.BaseStylesMap("Row Header").StyleInfo.TextColor = Color.Orange
...
+Custom BaseStyles can be defined in GridControl by using the [GridBaseStyle](hhttps://help.syncfusion.com/cr/windowsforms/Syncfusion.Windows.Forms.Grid.GridBaseStyle.html). After defining a BaseStyle, it has to be added to the [BaseStylesMap](https://help.syncfusion.com/cr/windowsforms/Syncfusion.Windows.Forms.Grid.GridBaseStylesMap.html) so that it can be accessed in `GridStyleInfo` objects. Assign the defined base style to the desired cells by using the [BaseStyle](https://help.syncfusion.com/cr/windowsforms/Syncfusion.Windows.Forms.Grid.GridStyleInfo.html#Syncfusion_Windows_Forms_Grid_GridStyleInfo_BaseStyle) property in the `GridStyleInfo`.
```
Comment (Girija-Rajendran) [inline WindowsForms/Grid-Control/Cell-Style-Architecture.md:329]: image

File: WindowsForms/DataGrid/InteractiveFeatures.MD (hunk 3 above)
Comment (SellineRita) [reply WindowsForms/DataGrid/InteractiveFeatures.MD:653]: Reverted changes

File: WindowsForms/DataGrid/InteractiveFeatures.MD (hunk 2 above)
Comment (SellineRita) [reply WindowsForms/DataGrid/InteractiveFeatures.MD]: Updated correct link.

File: WindowsForms/Grid-Control/Cell-Style-Architecture.md [hunk 5]
```diff
@@ -85,7 +85,7 @@ Me.gridControl1(3,2).CellValue = obj
...
+[ImageList](https://learn.microsoft.com/en-us/dotnet/api/system.windows.forms.imagelist?view=windowsdesktop-7.0&viewFallbackFrom=net-5.0) property holds [Systems.Windows.Forms.ImageList](https://learn.microsoft.com/en-us/dotnet/api/system.windows.forms.imagelist?view=windowsdesktop-7.0&viewFallbackFrom=net-5.0). Generally the whole grid will store the ImageList in the parent `GridStyleInfo`. The images can be retrieved by using [ImageIndex](https://help.syncfusion.com/cr/windowsforms/Syncfusion.Windows.Forms.Grid.GridStyleInfo.html#Syncfusion_Windows_Forms_Grid_GridStyleInfo_ImageIndex) property and can set on cell-by-cell basis.
```
Comment (Girija-Rajendran) [inline WindowsForms/Grid-Control/Cell-Style-Architecture.md:88]: is this correct?

Comment (SellineRita) [reply WindowsForms/Grid-Control/Cell-Style-Architecture.md]: Updated correct link.

**Review by Girija-Rajendran — APPROVED** [review]

**Pull request merged**